### PR TITLE
BF+ENH+DOC: An actually working motion QC plot command

### PR DESCRIPTION
**Pull request by hanke**
Not sure why the previous one was merged in a broken state. But I took the freedom to make it a bit more flexible.

### CODE REVIEW

File: mvpa2/cmdline/cmd_ofmotionqc.py [hunk 1]
```diff
@@ -53,63 +91,85 @@ def run(args):
...
-    segment_sizes = [len(d[0]) for d in data]
+    segment_sizes = [len(d[d.keys()[0]]) for d in data]
+
+    if not args.relative is None:
+        # recode per-subject estimates wrt their particular reference
+        ref = {subj: d[args.relative[1]]
+               for subj, d in data[args.relative[0]].iteritems()}
+        for d in data:
+            for subj in d:
+                if subj in d:
```
Comment (yarikoptic) [inline mvpa2/cmdline/cmd_ofmotionqc.py]: not obvious to me: how can subj not be in d if we loop through it?

File: mvpa2/cmdline/cmd_ofmotionqc.py [hunk 2]
```diff
@@ -53,63 +91,85 @@ def run(args):
     from mvpa2.datasets.sources.openfmri import OpenFMRIDataset
```
Comment (yarikoptic) [inline mvpa2/cmdline/cmd_ofmotionqc.py]: could there be at least a smoke test for this now that we have a sample openfmri 1 subj tutorial_data?

File: mvpa2/cmdline/cmd_ofmotionqc.py [hunk 3]
```diff
@@ -53,63 +91,85 @@ def run(args):
...
-    #pl.ylim((0,5))
-    pl.title('rotation')
-    pl.ylabel('estimate L2-norm in deg')
     pl.xlabel('time in fMRI volumes')
 
     if args.savefig is None:
         pl.show()
     else:
-        pl.savefig(args.safefig)
+        pl.savefig(args.savefig)
```
Comment (yarikoptic) [inline mvpa2/cmdline/cmd_ofmotionqc.py]: unittest! unittest! ;)

File: mvpa2/cmdline/cmd_ofmotionqc.py (hunk 2 above)
Comment (hanke) [inline mvpa2/cmdline/cmd_ofmotionqc.py]: The moment we have a multi-subject demo dataset there will be a test.

**Comment by hanke** [comment]
For the record: the test failure was a network issue.

**Comment by hanke** [comment]
Merging this to have a working command. More beauty with next iteration.

**Pull request merged**